src/controllers/logs.go

```diff
@@ -0,0 +1,70 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"go.uber.org/zap"
+	"io"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/dana-team/platform-backend/src/utils"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	cappLabel = "rcs.dana.io/parent-capp=%s"
```

Comment:
Construct using this:

https://github.com/dana-team/platform-backend/blob/1b400331fd0429f6c54f46843bcb77341c46918c/src/utils/labels.go#L8